jaspersoft-containers/K8s/jrs/README.md

```diff
@@ -207,10 +207,10 @@ These details are stored in Kubernetes secrets and used as environment variables
       AUDIT_DB_NAME:  audit-db-name
       AUDIT_DB_USER_NAME: audit-db-user-name
       AUDIT_DB_PASSWORD: audit-password
-      
-4. Build the docker images for TIBCO JasperReports® Server, and Scalable Query Engine (see the [Docker JasperReports Server readme](../../Docker/jrs#readme) and [Docker Scalable Query Engine readme](../../Docker/scalableQueryEngine#readme) ).
-5. Generate the keystore and copy it to the `k8s/jrs/helm/secrets/keystore` folder, see here for [Keystore Generation ](../../Docker/jrs#keystore-generation).
-6. Copy the TIBCO JasperReports® Server license to the `k8s/jrs/helm/secrets/license` folder.
+
+3. Build the docker images for TIBCO JasperReports® Server, and Scalable Query Engine (see the [Docker JasperReports Server readme](../../Docker/jrs#readme) and [Docker Scalable Query Engine readme](../../Docker/scalableAdhocWorker#readme) ).
```

Comment:
link is wrong I think, should be as before - (../../Docker/scalableQueryEngine#readme)